src/libs/ReportActionsUtils.js

```diff
@@ -219,10 +219,11 @@ function shouldReportActionBeVisible(reportAction, key) {
         return false;
     }
 
-    // All other actions are displayed except deleted, non-pending actions
+    // All other actions are displayed except thread parents, deleted, or non-pending actions
     const isDeleted = isDeletedAction(reportAction);
     const isPending = !_.isEmpty(reportAction.pendingAction);
-    return !isDeleted || isPending;
+    const isDeletedParent = lodashGet(reportAction, ['message', 0, 'isDeletedParentAction'], false);
+    return !isDeleted || isPending || isDeletedParent;
```

Comment:
Where `isDeletedParentAction` come from? I don't see it set anywhere.

My current assumption is that this come from the backend, if so we are missing the optimistic data for this, but I guess the delete thread functionality is going to be implemented in a follow up PR, would that be right?

Also should not the variable here be `isDeletedParentReportAction` for consistency.